webui.sh

```diff
@@ -56,7 +56,7 @@ then
 fi
 
 # this script cannot be run as root by default
-can_run_as_root=0
+can_run_as_root=1
```

Comment:
Why is this needed?

Reply:
The directories get added as root user. Did not have enough time to debug and figure out assigning as a user, hence running the script as root. Will figure this out in subsequent PRs.